### PR TITLE
add couchpotato plugin

### PR DESCRIPTION
**Pull request by midraal**
This plugin allows retrieval of the couchpotato wanted movie list

example usage:
```
   queue_movies_couchpotato:
     couchpotato:
       base_url: http://localhost
       port: 5050
       api_key: <your key here>
     accept_all: yes
     movie_queue: add
```

### CODE REVIEW

File: flexget/plugins/input/couchpotato.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    Options base_url, port and api_key are required.
+    """
+        url = '%s:%s/api/%s/movie.list?status=active' % (config['base_url'], config['port'], config['api_key'])
+        json = task.requests.get(url).json()
+        entries = []
+        for movie in json['movies']:
+            if movie['status'] == 'active':
+              title = movie["title"]
+              entry = Entry(title = '%s' % (title),
```
Comment (cvium) [inline flexget/plugins/input/couchpotato.py]: We aim to have all new code PEP8 compliant, which states that keyword arguments shouldn't have any spaces ie. it should be `title='%s'`. Is it even necessary to do it like that? Wouldn't it work with `title=title`?

**Comment by midraal** [comment]
it now runs through the pep8 script without errors

File: flexget/plugins/input/couchpotato.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import requests
+
+from flexget import plugin
+from flexget.event import event
+from flexget.entry import Entry
+
+log = logging.getLogger('couchpotato')
+
+
+class CouchPotato(object):
```
Comment (ianstalk) [inline flexget/plugins/input/couchpotato.py]: Usage info should go here as a docstring. This is what's printed when the --doc option is used from the command line.

**Comment by gazpachoking** [comment]
Does the couchpotato api provide any movie ids? If so, can we add them to the entries with appropriate names? (imdb_id, tmdb_id)

**Comment by gazpachoking** [comment]
Thanks! Want to make a wiki page for it?

**Pull request merged**